src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReportingService.kt

```diff
@@ -38,9 +40,11 @@ class ReportingService(
   }
 
   fun generateNdmisPerformanceReport() {
+    val outputDir = createTempDirectory("test")
     asyncJobLauncher.run(
       ndmisPerformanceReportJob,
       JobParametersBuilder()
+        .addString("outputPath", outputDir.pathString)
```

Comment:
What is the significance of this changes

Reply:
When david changed the job to require and outputPath for testing it was missed here. This is used to run the job through an endpoint to test with

Reply:
Ok thanks @rgforsyth